hopglass-server.js

```diff
@@ -480,9 +480,24 @@ function getFfmapJson(stream) {
 
 function getMetrics(stream) {
   getData()
-  var save = (n, id, stream, what, where) => {
-    if (_.has(n, what))
-      stream.write((where ? where : what.replace(/\./g, '_')) + id + ' ' +  _.get(n, what) + '\n')
+  var save = (n, stream, labels, what, value) => {
+    var id = '{'
+    var first = true
+    for (var e in labels) {
+      if (first)
+        first = false
+      else
+        id += ','
+      id += e + '="' + labels[e] + '"'
+    }
+    id += "}"
```

Comment:
I guess this is the way it should be. The reason it worked before was probably just that js compensated for my language ignorance :)